extensions/apps/antenna/src/extensions/beam-editor/beam-editor.tsx

```diff
@@ -105,6 +115,29 @@ export const BeamEditor: React.FC = () => {
     }
   }, [blocksInUse]);
 
+  useEffect(() => {
+    let navigationSubscribe: () => void;
```

Comment:
this is `navigationUnsubscribe` because `routing.cancelNavigation` returns a function that when it's called, it will unsubscribe from the event `single-spa:before-routing-event`